setup.py

```diff
@@ -55,6 +53,7 @@
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, <4",
```

Comment:
This will explicitly drop 2.6 and 3.0-3.3, which both receive ~5 downloads per day so I don't think are very critical? urllib3 and Requests have both not supported these versions for years now.

Reply:
awesome, explicit is better than implicit!